library/alloc/src/slice.rs

```diff
@@ -178,10 +178,15 @@ impl<T> [T] {
     /// This sort is stable (i.e., does not reorder equal elements) and *O*(*n* \* log(*n*))
     /// worst-case.
     ///
-    /// If the implementation of [`Ord`] for `T` does not implement a [total order] the resulting
-    /// order of elements in the slice is unspecified. All original elements will remain in the
-    /// slice and any possible modifications via interior mutability are observed in the input. Same
-    /// is true if the implementation of [`Ord`] for `T` panics.
+    /// If the comparison function compare does not implement a [total order], the function may
```

Comment:
This is on .sort(), so I'm not sure why we're saying "comparison function compare" -- the `Ord` impl wording seems better?

Reply:
🤦🏻 I edited the wrong doc-comment. Apologies! But I suppose both `sort` and `sort_by` should have analogous changes.